crates/nu-cli/tests/completions/mod.rs

```diff
@@ -428,6 +470,16 @@ fn command_open_with_filecompletion() {
         ".hidden_folder/".to_string(),
     ];
 
+    match_suggestions(expected_paths, suggestions);
+
+    let target_dir = "open custom_completion.";
+    let suggestions = completer.complete(target_dir, target_dir.len());
+
+    #[cfg(windows)]
+    let expected_paths: Vec<String> = vec!["custom_completion.nu".to_string()];
+    #[cfg(not(windows))]
+    let expected_paths: Vec<String> = vec!["custom_completion.nu".to_string()];
```

Comment:
looks like the two `cfg` branches are the same?
```suggestion
    let expected_paths: Vec<String> = vec!["custom_completion.nu".to_string()];
```

Reply:
Oh, good catch!  I made some fool copy paste.